pdr_backend/pdr_dashboard/util/helpers.py

```diff
@@ -0,0 +1,27 @@
+import dash
+
+
+def toggle_modal_helper(
+    ctx,
+    selected_rows,
+    is_open_input,
+    safe_trigger_ids=None,
```

Comment:
I just remembered here that safe trigger ids completely depend on the id, I will adjust these.